docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -4,39 +4,35 @@ and pull requests should be submitted there:
 include::_attributes.adoc[]
 :diataxis-type: tutorial
 :categories: security,getting-started
 :topics: security,authentication,basic-authentication,http
 :extensions: io.quarkus:quarkus-vertx-http,io.quarkus:quarkus-elytron-security-jdbc,io.quarkus:quarkus-elytron-security-ldap,io.quarkus:quarkus-security-jpa-reactive
 
-Get started with Quarkus Security by securing your Quarkus application endpoints with the built-in Quarkus xref:security-basic-authentication.adoc[Basic authentication] and the Jakarta Persistence identity provider and enabling role-based access control.
+Get started with Quarkus Security by securing your Quarkus application endpoints with the built-in Quarkus xref:security-basic-authentication.adoc[Basic authentication] and the Jakarta Persistence identity provider, enabling role-based access control.
 
-The Jakarta Persistence `IdentityProvider` verifies and converts a xref:security-basic-authentication.adoc[Basic authentication] user name and password pair to a `SecurityIdentity` instance which is used to authorize access requests, making your Quarkus application secure.
+The Jakarta Persistence `IdentityProvider` verifies and converts a xref:security-basic-authentication.adoc[Basic authentication] user name and password pair to a `SecurityIdentity` instance, which is used to authorize access requests, making your Quarkus application secure.
 
 For more information about Jakarta Persistence, see the xref:security-jpa.adoc[Quarkus Security with Jakarta Persistence] guide.
 
-This tutorial prepares you for implementing more advanced security mechanisms in Quarkus, for example, how to use the OpenID Connect (OIDC) authentication mechanism.
+This tutorial prepares you to implement more advanced security mechanisms in Quarkus, for example, how to use the OpenID Connect (OIDC) authentication mechanism.
 
 == Prerequisites
 
 include::{includes}/prerequisites.adoc[]
 
-== What you will build
+== Building your application
 
-To demonstrate different authorization policies, the steps in this tutorial guide you through building an application that provides the following endpoints:
+This tutorial provides detailed steps for creating an application with endpoints that illustrate various authorization policies:
 
 [cols="20%,40% ",options="header"]
 |===
 |Endpoint | Description
-|`/api/public`| The `/api/public` endpoint can be accessed anonymously.
-|`/api/admin`| The `/api/admin` endpoint is protected with role-based access control (RBAC).
-Only users granted with the `admin` role can access it.
-At this endpoint, the `@RolesAllowed` annotation enforces the access constraint declaratively.
-|`/api/users/me`| The `/api/users/me` endpoint is protected by RBAC.
-Only users that have the `user` role can access the endpoint.
-This endpoint returns the caller's username as a string.
+|`/api/public`| Accessible without authentication, this endpoint allows anonymous access.
```

Comment:
This refactoring reads unusual and is looks complex, `access is regulated declaratively`, this is a bit too complex, IMHO the current text is simple and clear

Reply:
Or may be I need to read it a few times, reads better after a 2nd try :-).